app/wedding/tests/test_guest_api.py

```diff
@@ -0,0 +1,74 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from core.models import Wedding
+from core.models import Guest
+
+from wedding.serializers import GuestSerializer
+
+CREATE_WEDDING_URL = reverse('wedding:create')
+GUESTS_URL = reverse('wedding:guest-list')
+CREATE_GUEST_URL = reverse('wedding:create_guest')
```

Comment:
If we want more customized urls that follow the convention we're used to, we can probably look into getting rid of reverse.

Reply:
Great point! We should definitely look into that!